arbor/fvm_layout.cpp

```diff
@@ -1269,6 +1295,8 @@ fvm_mechanism_data fvm_build_mechanism_data(
                 throw cable_cell_error("revpot mechanism for ion "+ion+" does not write this reversal potential");
             }
 
+            M.ions[ion].revpot_written = true;
```

Comment:
This is always set to true from my reading if the code. Is that meant to be?

Reply:
Yes, that's why this writes directly to `M.ions` and does not go through a intermediary. Or did I misread your comment?